packages/nx/src/tasks-runner/tasks-schedule.spec.ts

```diff
@@ -43,20 +44,20 @@ describe('TasksSchedule', () => {
         roots: ['lib1:build', 'app2:build'],
       };
       const workspace: Partial<Workspaces> = {
-        readExecutor() {
-          return {
-            schema: {
-              version: 2,
-              properties: {},
-            },
-            implementationFactory: jest.fn(),
-            batchImplementationFactory: jest.fn(),
-          } as any;
-        },
         readNxJson() {
           return {};
         },
       };
+      (executorUtils as any).getExecutorInformation = jest
+        .fn()
+        .mockReturnValue({
+          schema: {
+            version: 2,
+            properties: {},
+          },
+          implementationFactory: jest.fn(),
+          batchImplementationFactory: jest.fn(),
+        });
```

Comment:
Does jest.spyOn not work here? It should avoid needing the cast to `any`

Reply:
changed